### ISSUE TITLE
How to add to Joplin?

### ISSUE THREAD

**Issue by chandler150** [DanteCoder/JoplinPluginFolderNoteLinks#1]
Having trouble adding this plugin as it is not found in the search.

I suppose I could download to a folder and specify that folder as a path in the advanced section of the plugins settings page, but that'd just be my guess. What did you do?

**Comment by DanteCoder** [comment]
Yes, the plugin is not released because it's unfinished, and I don't know if I would release it soon. Right now it's just a test.

If you want to try it, you have to clone the repo and checkout the dev branch, then specify the path in Joplin.
